### PR TITLE
Fix y offset of sloped castle walls

### PR DESCRIPTION
**Pull request by Gymnasiast**
Good Knight Park before:
Good Knight Park 2024-04-04 21-00-32
and after:
Good Knight Park 2024-04-04 20-59-40

Sloped castle wall on top of sloped sandstone wall, before:
afbeelding
and after:
afbeelding

Crazy Castle perimeter wall, before:
image
and after:
image

### CODE REVIEW

**Comment by ZehMatt** [comment]
Just to make sure, does it render like that in the original too?

**Comment by Gymnasiast** [comment]
Here is Good Knight Park in vanilla RCT:

image

**Comment by ZeeMaji** [comment]
For some very strange reason, the objects builder is lumping all of the wall objects into a single parkobj in the rct2 objects folder as  "rct2.scenery_wall.wallbadm.parkobj"

**Comment by ZehMatt** [comment]
> Here is Good Knight Park in vanilla RCT:
> 
> image

They seem to be fine here? Where is the difference coming from, thats the same asset right?

File: build.mjs [hunk 1]
```diff
@@ -86,12 +86,16 @@ function isImageLgxCompatible(image) {
             return false;
         }
     } else {
-        return true;
+        if (image.hasOwnProperty('path')) {
+            return true;
+        } else {
+            return false;
+        }
```
Comment (AaronVanGeffen) [inline build.mjs:93]: We could do without the branch here, right?
```suggestion
        return image.hasOwnProperty('path'));
```

Comment (Gymnasiast) [reply build.mjs:93]: I guess.

**Comment by ZeeMaji** [comment]
Objects seem to be building fine now, and as for the fix itself the gaps between the sloped walls and the floor are gone and look much better now. Nicely done.

**Review by ZeeMaji — APPROVED** [review]

**Pull request merged**